addons/http_routing/models/ir_http.py

```diff
@@ -587,7 +587,7 @@ def _get_exception_code_values(cls, exception):
             code = exception.code
 
         values.update(
-            status_message=werkzeug.http.HTTP_STATUS_CODES[code],
+            status_message=werkzeug.http.HTTP_STATUS_CODES.get(code,''),
```

Comment:
Hey,

Did you have a linter ? Missing whitespace after the comma... One line patch, one lint error :100:  

Else, what is the step to reproduce it ? 
Because you point 2 opw, but without way to reproduce it if I'm not wrong. Just 2 migration with one problem with oauth module.

From what I can see in the code,  We should always have a code (or custom, or [werkzeug](https://werkzeug.palletsprojects.com/en/1.0.x/exceptions/)... So it will be interesting to investigate the use case to fix probably the root cause, instead to hide the bug under the carpet :)

And please, ask in case of doubts.

//cc @odony 
Most of PR in master are in dev. While in v13, we know that it is trivial diff and not wip pr.
What you think about backport codeowner in 13.0 ?

Reply:
> Most of PR in master are in dev. While in v13, we know that it is trivial diff and not wip pr.

Sure, we just wanted to try it out in master first. We can start working on a PR for 13.0 now that we have some experience with it ;)